terraform/cloud-platform/main.tf

```diff
@@ -127,18 +127,13 @@ resource "auth0_client" "components" {
   app_type    = "regular_web"
 
   callbacks = [
-    "https://prometheus.apps.${local.cluster_base_domain_name}/oauth2/callback",
-    "${terraform.workspace == local.live_workspace ? format("https://prometheus.%s/oauth2/callback", local.live_domain) : ""}",
-    "https://alertmanager.apps.${local.cluster_base_domain_name}/oauth2/callback",
-    "${terraform.workspace == local.live_workspace ? format("https://alertmanager.%s/oauth2/callback", local.live_domain) : ""}",
+    "${terraform.workspace == local.live_workspace ? format("https://prometheus.%s/oauth2/callback", local.live_domain) : "https://prometheus.apps.${local.cluster_base_domain_name}/oauth2/callback"}",
```

Comment:
What do you think about changing:
```
"${terraform.workspace == local.live_workspace ? format("https://prometheus.%s/oauth2/callback", local.live_domain) : "https://prometheus.apps.${local.cluster_base_domain_name}/oauth2/callback"}"
```
to:
```
"${format("https://prometheus.%s/oauth2/callback", terraform.workspace == local.live_workspace ? local.live_domain : local.cluster_base_domain_name}",
```
I believe it look much cleaner and is easier to read.

Reply:
As an afterthought, we could subsequently make `${terraform.workspace == local.live_workspace ? local.live_domain : local.cluster_base_domain_name}` another `local` variable so that it only evaluates once.

Reply:
neat, shortened